plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/rest/TestIcebergVendingRestCatalogConnectorSmokeTest.java

```diff
@@ -91,7 +91,8 @@ protected QueryRunner createQueryRunner()
             throws Exception
     {
         Network network = Network.newNetwork();
-        minio = closeAfterClass(Minio.builder().withNetwork(network).build());
+        // Use new MinIO version because Iceberg enables strong integrity checks
+        minio = closeAfterClass(Minio.builder().withImage("minio/minio:RELEASE.2025-01-20T14-49-07Z").withNetwork(network).build());
```

Comment:
this will break iceberg in storage like ECS. Can we force iceberg not to use storng integrity checks?

Reply:
I reported the issue at Iceberg Slack today: https://apache-iceberg.slack.com/archives/C03LG1D563F/p1739434861646329?thread_ts=1739417985.509649&cid=C03LG1D563F